docs/source/reference/swe.rst

```diff
@@ -16,3 +16,25 @@ The L0 code to decommutate the CCSDS packet data can be found below.
     :recursive:
 
     l0.decom_swe
+
+The L1A code to unpack electron counts can be found below.
+
+.. autosummary::
+    :toctree: generated/
+    :template: autosummary.rst
+    :recursive:
+
+    l1a.swe_l1a
+    l1a.swe_science
+
+The L1A code to process electron counts to rates and then
```

Comment:
```suggestion
The L1B code to process electron counts to rates and then
```
Out of curiosity are we capitalizing both "L" and "B" here, or are we calling them "L1b" or "l1b"?

Reply:
I'm not particular about which version to choose, my only preference is for consistency if we can meet that. I think filenames will be lowercase `l1b`, but maybe if we are writing in sentences we will write `L1B` with all caps like you have?